src/labels/CustomLabels.labels

```diff
@@ -1115,6 +1115,54 @@ To check your GAU Allocation settings, go to Donations | GAU Allocations.</value
         <value>An error occurred during the process.  The following status was the first reported error:</value>
     </labels>
     <labels>
+        <fullName>bdiBehaviorBestMatchOrCreate</fullName>
+        <categories>Data Importer</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>settings label for BestMatchOrCreate behavior</shortDescription>
+        <value>First Match or Create - Import a record if it matches an existing record; create a new record if no match found.</value>
```

Comment:
Thanks, sorry for the mention spam, I wanted to be sure all the strings were reviewed since we had just discussed it in standup.